gdc_client/upload/client.py

```diff
@@ -410,7 +415,9 @@ def delete(self):
             if r.status_code == 204:
                 log.info("Delete file {}".format(self.node_id))
             else:
-                log.warning("Fail to delete file {}: {}".format(self.node_id, r.content))
+                log.warning(
+                    "Fail to delete file {}: {}".format(self.node_id, r.content)
+                )
```

Comment:
Want to use f-strings because we're on python3 now?

Reply:
so this was the thing i was thinking of during the pre-commit meeting right now, basically i had f-strings here, and then when i pushed, it failed the build check because it is running against python 3.5 also
https://github.com/NCI-GDC/gdc-client/blob/develop/.travis.yml#L15
so I reverted to the old way of formatting